src/operator/tensor/broadcast_reduce_op.h

```diff
@@ -411,6 +447,149 @@ void ReduceAxesCompute(const nnvm::NodeAttrs& attrs,
   ReduceAxesComputeImpl<xpu, reducer, normalize>(attrs, ctx, inputs, req, outputs, small);
 }
 
+template <int req, int axis>
+struct SumCsrKernel;
+
+template <int req>
+struct SumCsrKernel<req, 0> {
+  template <typename RType, typename IType, typename DType>
+  MSHADOW_XINLINE static void Map(int j, DType* out_data,
```

Comment:
Add a comment here explaining the meaning of `j` so that it's easy to understand how this is parallelized.

Reply:
Added comments